# The qoauth frameworks in src/src.pro doesn't really work - use the unix part
# Don't use lib64
--- src/src.pro.orig	2010-08-01 09:08:59.000000000 -0500
+++ src/src.pro	2014-05-04 23:49:33.000000000 -0500
@@ -40,7 +40,7 @@
 features.files = ../oauth.prf
 docs.files = ../doc/html
 
-macx {
+macxNOFRAMEWORKS {
     CONFIG += lib_bundle
     QMAKE_FRAMEWORK_BUNDLE_NAME = $$TARGET
     CONFIG(debug, debug|release): CONFIG += build_all
@@ -62,7 +62,7 @@
     pkgconfig.files = qoauth.pc
     
     contains(QMAKE_HOST.arch, x86_64) {
-      target.path = $${INSTALL_PREFIX}/lib64
+      target.path = $${INSTALL_PREFIX}/lib
     } else {
       target.path = $${INSTALL_PREFIX}/lib
     }
